js/panel_controller.js

```diff
@@ -19,11 +19,19 @@
 
     window.drawAttention = function() {
         if (inboxOpened && !inboxFocused) {
+            if( window.keepClear ) {
```

Comment:
Our code style in this project does whitespace a little differently. Take a look at the line above. I'd like to see this line like this: `if (window.keepClear) {`